tests/sagemaker-tests/test_sagemaker_args.py

```diff
@@ -86,3 +90,144 @@ def test_parse_complex_string_with_quotes():
         "feat-name": "ntype0:feat0 ntype1:feat1",
     }
     assert dict(parse_unknown_gs_args(args)) == expected
+
+
+def test_get_realtime_infer_argparser():
```

Comment:
Not sure whether we need unitests for get_realtime_infer_parser

Reply:
there are a few place required for unit tests, e.g., the format of `model_name`.